### PR TITLE
improve lyric centering documentation

### LINKED ISSUE #899: bug in centering <sp>ae'</sp>
This example:

```
\documentclass{article}
\usepackage{fontspec}
\usepackage{gregoriotex}
\begin{document}
\gabcsnippet{(c4) gr<sp>'ae</sp>(c) grǽ(c)}
\end{document}
```

shows a difference in the notes alignment between `grǽ` and `gr<sp>'ae</sp>` which shouldn't exist

### PR DESCRIPTION
**Pull request by eroux**
Documents #899 and #838. I think we can close #899 once this is merged.

### CODE REVIEW

File: doc/Gabc.tex [hunk 1]
```diff
@@ -286,19 +286,40 @@ \subsection{Simple Slurs}
...
 
 The default rules built into Gregorio for \texttt{vowel} mode are for
 Ecclesiatical Latin and work fairly well (though not perfectly) for
 other languages (especially Romance languages).  However, Gregorio
 provides a gabc \texttt{language} header which allows the language of
-the lyrics to be set.  Gregorio will look for a file named
+the lyrics to be set. The default is Latin.
+
+Special characters (input with \texttt{<sp>}) or verbatim text (\texttt{<v>})
+count as consonnants, so you have to force centering around them, for example
```
Comment (henryso) [inline doc/Gabc.tex]: Spelling: "consonnant" should be "consonant" (also two lines below).

**Comment by henryso** [comment]
Other than that, it looks OK to me.

**Comment by eroux** [comment]
Thanks!

**Comment by rpspringuel** [comment]
I agree.  Looks good.

**Pull request merged**